ramalama/common.py

```diff
@@ -38,7 +38,11 @@ def exec_cmd(args):
     if x:
         print(*args)
 
-    return os.execvp(args[0], args)
+    try:
+        return os.execvp(args[0], args)
+    except Exception:
+        print(f"os.execvp({args[0]}, {args}) failed")
```

Comment:
Print to stderror perror?

Reply:
will change

Reply:
I actually added a python perror-ish function called perror here. Like you, I tend to like these C-style functions

Reply:
Wait you implemented one already, nevermind, using that